docs/02-quick-start/README.mdx

```diff
@@ -8,4 +8,43 @@ import DocCardList from '@theme/DocCardList';
 
 This section provides instructions to quickly start an Ozone instance on your local machine and run basic operations against it.
 
+<div className="container">
+  <div className="row">
+    <div className="col col--6 margin-bottom--lg">
+      <a className="card padding--lg card--full-height" href="installation">
```

Comment:
Sometimes when I clicked both "Installation" and "Reading & Writing Data", I encountered "Page Not Found" since I was routed to `/docs/installation` instead `docs/quick-start/installation/`. 

Can be reproduced by 
1. Going to the home page
2. Press "Get Started"
3. Press "Installation" or "Reading & Writing Data".

If I access the Quick Start from the side panel, it works fine.

Maybe we need to use an absolute path for the "href".